translathon.qmd

```diff
@@ -2,12 +2,170 @@
 
-Participants completed an optional feedback survey collecting insights on their experiences, challenges faced, and suggestions for improvement.
\ No newline at end of file
+Participants completed an optional feedback survey collecting insights on their experiences, challenges faced, and suggestions for improvement.
+
+The community call provided guidance on how to get started with translation contributions and different translations and localization efforts that exist for a human language in the community.
+
+## Mentors and Editors
+
+The main goal is traslathon events is to ensure participants have a good experience:
+
+-   learning how to contribute to translations,
+-   gaining confidence in the process, and
+-   building motivation to continue contributing beyond the event.
+
+To achieve this, traslathons rely on two key roles:
+
+-   **Mentors:** contributors familiar with the translation process, tools, and community guidelines.
+    They help participants get started, troubleshoot issues, and guide them through the workflow.
+
+-   **Editors:** experienced translators responsible for reviewing and maintaining the quality and consistency of translations.
+    Editors check terminology, style, and adherence to the community’s translation agreements.
+
+Mentors and Editors sign up to participate beforehand and review the [translation process and rules](https://translationguide.ropensci.org), and the translation that need contribution.
+They are preferably people who have already contributed, as reviewers or editors, to rOpenSci's translations and know how the tools used in the process work and are also aware of the translation agreements of the language in which they are involved.
+
+Because preparing and reviewing translations requires additional effort, we suggest offering mentors and editors an honorarium to recognize their time and contributions.
+
+> In our 2024 pilot, the traslaton was hosted in Portuguese in conjunction with LatinR, where many members of the Latin American R community gathered.
+> We focus the translation effort to one resource: rOpenSci Packages: Development, Maintenance, and Peer Review.
+> NumFocus provide the funding to pay mentors and editors a \$200 USD honorarium for participating in the traslathons.
+
+
+## Timeline
+
+See [Events Overview](events-overview.html) for the full timeline of paired events.
+
+The timeline is setup for a single event, but repeat events are easy to insert after the first three tasks.
+
+|  |  |  |
+|-------------------------|-----------------------|-------------------------|
+| **Weeks Before** | **Task** | **Platform** |
+| 10 | Choose date for traslathon or an event to joint organization. |  |
+| 8 | Open call for mentors/editors at hands-on events | [Blog](https://ropensci.org/blog/) / Registration/ [Social](https://hachyderm.io/@rOpenSci) (include Slack workspaces) |
```

Comment:
I added a template for social media posts, let's link to that and keep this simple.
```suggestion
| 8 | Open call for mentors/editors at hands-on events | [Blog](https://ropensci.org/blog/) / Registration/ [Social](templates_social.html) |
```